sql/encryption.cc

```diff
@@ -187,6 +187,11 @@ static uint scheme_get_key(st_encryption_scheme *scheme,
   return rc;
 }
 
+/** Run encryption or decryption on a block.
+ * `i32_1`, `i32_2`, and `i64` are used to create the initialization vector
+ * \invariant `src` must be valid for `slen`
```

Comment:
I suppose it should be @invariant — everywhere else in the codebase doxygen commands start from `@`